data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+	`	She seems surprised that you asked and responds, "Oh, they're a sort of rebel Hai group. If you don't know you should go visit Cloudfire in the Wah Ki system up north, or maybe even Darkcloak if you dare. A short visit will explain it far better than I ever could."`
+		decline
+
+
+conversation "What is going on in Hai space? (Met the Unfettered.)"
+	`Landing in Hai space draws a much different type of attention than expected. Only recently you've begun to acclimate yourself to getting positive attention. As a war hero you're something of a minor celebrity now back in human space, but here suddenly mothers are keeping tight grips on their children, and passers-by look upon you with a hint of wariness.`
+	`	You realize that the few humans who have settled here among the Hai are all wearing pink at the moment, and when you catch the eye of one lady who looks friendly you greet her to ask a few questions.`
+	choice
+		`	"Why does everyone seem so afraid?"`
+		`	"Why are you all wearing pink?"`
+			goto pink
+	`	She shakes her head and wrinkles her nose with a look of distaste. "Some marauder came here just yesterday, posing as a trader to investigate the rumors. We thought the Hai had means of keeping them out, but this one snuck in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
+	`	She sighs deeply and says, "Everyone is just a bit shaken and distrustful at the moment."`
+	`	"Oh that's awful!" you say.`
+	`	She nods and forces a little bit of false cheer into her voice, "This is why we're all wearing pink, just a little show of local solidarity."`
+		goto main
+	label pink
+	`	A sad smile crosses her face and her expression goes sombre as she explains, "A pirate marauder snuck down onto the planet recently, stole a Geocoris, and kidnapped a number of Hai, probably into slavery. Everyone is a bit shaken up so the locals are wearing pink in solidarity, and so we're not mistaken for a security risk."`
+	label main
+	`	"Unfortunately, it's taking a while for them to redistribute an appropriately resourced security force thanks to the Unfettered."`
+	`	Your conversation is interrupted suddenly by a small group of somewhat official-looking Hai.`
+	`	"Are you Captain <last>?" one of them asks you. "Don't worry, you're not in trouble or anything."`
+	`	You nod your affirmation.`
+	`	"Your reputation precedes you, Captain. If you have any stake in the current information crisis, please head directly to Hai-home. Someone would very much like to talk to you. As a person of note, you are presently being treated as an Envoy." They leave without waiting for your agreement. Then again, you did come here to see what was going on.`
+		decline
+
+
+# Six possibilities for origin and destination, to work around bug 1938
+# Symbols RS=Republic or Syndicate FW = Free Worlds
+# H=met Hai, U=met Unfettered
+
+mission "ZZ Event Hai Secret Leaks: RSHU"
```

Comment:
`	You suspect it might be time to go visit your Hai friends again.`
`	This is, of course, ridiculous... but maybe it would be good to go investigate this for yourself. You listen in to collect the relevant details for future reference.`

Not sufficient indicators of an objective?

Reply:
Oh _that_ right, yes, I see. That I can deal with.